lib/SQL/Translator/Producer/Oracle.pm

```diff
@@ -390,23 +391,9 @@ sub create_table {
                 push @field_defs, 'CONSTRAINT '.$index_name.' PRIMARY KEY '.
                     '(' . join( ', ', @fields ) . ')';
             }
-            elsif ( $index_type eq NORMAL ) {
-                $index_name = $index_name ? mk_name( $index_name )
-                    : mk_name( $table_name, $index_name || 'i' );
-                $index_name = quote($index_name, $qf);
-                push @index_defs,
-                    "CREATE INDEX $index_name on $table_name_q (".
-                        join( ', ', @fields ).
-                    ")$index_options";
-            }
-            elsif ( $index_type eq UNIQUE ) {
-                $index_name = $index_name ? mk_name( $index_name )
-                    : mk_name( $table_name, $index_name || 'i' );
-                $index_name = quote($index_name, $qf);
-                push @index_defs,
-                    "CREATE UNIQUE INDEX $index_name on $table_name_q (".
-                        join( ', ', @fields ).
-                    ")$index_options";
+            elsif ($index_type eq NORMAL or $index_type eq UNIQUE) {
+                warn "CAlling create index with options: " . Dumper($options) . "\n";
```

Comment:
"CAlling" typo

Reply:
My bad, that warn was there for debugging. I missed it when I was cleaning up. I'll take that out.

Reply:
That is from an old commit.